docs/main/Solutioning/backup-and-restore/kasten-br-fs.md

```diff
@@ -0,0 +1,338 @@
+---
+id: kastenbrfs
+title: Kasten Backup and Restore using Replicated PV Mayastor Snapshots - FileSystem
+keywords: 
+  - Kasten Backup and Restore using Replicated PV Mayastor Snapshots - FileSystem
```

Comment:
The keywords, titles and description needs to include OpenEBS for the search engines to source these documents. Same for Velero guides as well.

Reply:
I've added the term "OpenEBS" in the keywords. Although, the search engine renders the documents even without the term OpenEBS.